Cargo.toml

```diff
@@ -55,6 +55,8 @@ trust-dns-resolver = { version = "0.21.2", optional = true }
 portpicker = { version = "0.1.1", optional = true }
 num-bigint-dig = "0.8.1"
 tracing = { version = "0.1.37" }
+
+[target.'cfg(not(wasm))'.dependencies]
 rustls = { version = "0.20.7", features = ["dangerous_configuration"] }
```

Comment:
I think you can target windows directly here

Reply:
Yes. It was my first try to fix it. I came up with a better idea of how to properly fix it and I'm working on it now

Reply:
I added a separate feature `tsssp` in `sspi-rs` and `sspi-rs/ffi` that can only be enabled on Windows. The _TSSSP_ implementation is hidden behind the feature so sspi-rs can be built for the WASM target.  

Currently, CI builds sspi-rs with  `--all-features` flag on all systems which makes the build fail cause `tsssp` can be only enabled on Windows.  We should modify CI to build with the feature enabled only on Windows. I want to do it in a separate PR to not mix this PR, but I'm not sure if it's possible to merge a PR when there are some failed runners.